packages/open-truss/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as helloWorld } from './hello_world'
-export { parseYaml, stringifyYaml } from './utils/yaml'
+export * from './utils/yaml'
+export * from './utils/format'
```

Comment:
Can you update your editor to add a newline at the end of a file and if you don't have it already also remove whitespace from the end of lines.

Reply:
```suggestion
export * from './utils/format'

```